instances/atlas/parcellationEntity/JBA/JBA_Area-Op5.jsonld

```diff
@@ -0,0 +1,17 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Area-Op5",
```

Comment:
this should be OP5 (as it was OP1 - OP4)

Reply:
this also came directly from the json file hartmut passed us:

https://gin.g-node.org/FZJ-INM1-BDA/20210615-julichbrain-v2.9.0/src/master/workingdir/hbp-d000006_julichbrain-cytoatlas_dev/JulichBrainCytoAtlas/julichbrain-ontology_29.0.json#L1149

we can make the change now, if you give the OK.